src/util/fetchPokemonData.js

```diff
@@ -13,6 +14,17 @@ export const fetchPokemon = async (nameOrId) => {
   }
 };
 
+// make function for getting flavor text and pokemon color using speciesUrl
+export const fetchSpecies = async (nameOrId) => {
```

Comment:
Are you all using the Js wrapper? There are built-in functions and caching built-in.

Reply:
Yes Kenny, we are using JS wrapper, that script is in our index.html, and the file is in the root of our project.